src/cellfinder_core/detect/filters/volume/multiprocessing.py

```diff
@@ -68,18 +63,21 @@ def process(self):
             z_offset=self.setup_params[5],
         )
 
+    def process(self):
+        self.progress_bar = tqdm(
+            total=len(self.planes_paths_range), desc="Processing planes"
+        )
+
         while True:
             plane_id, plane, mask = self.data_queue.get()
-            logging.debug(f"Plane {plane_id} received for 3D filtering")
-            print(f"Plane {plane_id} received for 3D filtering")
-
             if plane_id is None:
                 self.progress_bar.close()
                 logging.debug("3D filter done")
                 cells = self.get_results()
                 self.output_queue.put(cells)
                 break
 
+            print(f"Plane {plane_id} received for 3D filtering")
```

Comment:
We might not need both, but can we keep the logging statements?

Reply:
👍 I'm just tidying this whole PR up now and to make sure I haven't accidentally changed any of the debug or print statements